### ISSUE TITLE
static masking for BERT or RoBERTa model

### ISSUE THREAD

**Issue by sgonzaloc** [huggingface/transformers#14284]
I would like to use static masking for Roberta and also BERT.

What I saw here is that the collector is always implmeneted like dynamic masking. https://github.com/huggingface/transformers/issues/5979

There're 2 issues with this. First, BERT is static masking so to be able to reproduce and run BERT like the original paper, we need to have it. Second, in MNLI the paper of Roberta says that is better the static option, so in some cases would be better and interesting to run it in static way. 

So in short, I really would like if somebody adds the missing option to be able to run static masking, and not like now that always is dynamic and we don't have any option for changing this

cc @sgugger

**Comment by sgugger** [comment]
You can do static masking in the function that preprocesses your dataset. The `data_collator` are only here for dynamic operations to apply at batching (dynamic paddings, dynamic masking etc.)
